### PR TITLE
Ignore mento contracts during release process

### PR DESCRIPTION
**Pull request by pahor167**
### Description

Update of release scripts to remove Mento from checks

### CODE REVIEW

File: packages/protocol/lib/compatibility/verify-bytecode.ts [hunk 1]
```diff
@@ -20,6 +20,19 @@ const ignoredContracts = [
   // These contracts are not in the Registry (before release 1)
   'ReserveSpenderMultiSig',
   'GovernanceApproverMultiSig',
+
+  // These are Mento contracts which we are no longer maintaining
+  'Exchange',
```
Comment (martinvol) [inline packages/protocol/lib/compatibility/verify-bytecode.ts]: Does this work if we try to check an old release? In some parts of the code we add the ignore only after a certain version, Does that make sense here?

Comment (pahor167) [reply packages/protocol/lib/compatibility/verify-bytecode.ts]: I updated the code to ignore those contracts only for version 9 and higher

**Review by soloseng — APPROVED** [review]
Looks good!

NIT: Would it be possible to have a single file that serves as reference for the list of contracts to ignore (`ignoredContractsV9`)? This would avoid repeating hardcoding of the list.

File: packages/protocol/scripts/truffle/make-release.ts [hunk 2]
```diff
@@ -225,19 +228,40 @@ export interface ProposalTx {
...
+    }
+
+    const contracts = readdirSync(join(argv.build_directory, 'contracts'))
+      .map((x) => basename(x, '.json'))
+      .filter(
+        (contract) =>
+          !ignoredContractsSet.has(contract) &&
+          !ignoredContractsSet.has(contract.replace('Proxy', ''))
+      )
+    console.log('ignoredContracts', ignoredContractsV9)
```
Comment (montera82) [inline packages/protocol/scripts/truffle/make-release.ts]: just double checking that this is really needed right?

**Review by montera82 — APPROVED** [review]
🚢

**Pull request merged**